README.md

```diff
@@ -33,19 +32,40 @@ public class YourService {
   }
 }
 ```
+### Convert MARC Authority record to Linked Data Graph
+```java
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.marc4ld.service.marc2ld.bib.MarcAuthority2ldMapper;
```

Comment:
Package name is `org.folio.marc4ld.service.marc2ld.authority`

`import org.folio.marc4ld.service.marc2ld.authority.MarcAuthority2ldMapper`

Reply:
right, fixed. Thanks!